src/common/browser-action.js

```diff
@@ -46,6 +44,8 @@ function _(str) {
  * the badge state.
  */
 function BrowserAction(chromeBrowserAction) {
+  var buildType = settings.buildType;
+
```

Comment:
This was moved here so that the current value of `settings.buildType` at the time `new BrowserAction` is called is used.